### DISCUSSION TITLE
混贴票据识别

### DISCUSSION THREAD

**Discussion by greatliu** [PaddlePaddle/PaddleOCR#12267] [Q&A]
## 背景

经过需求征集https://github.com/PaddlePaddle/PaddleOCR/issues/10334 和每周技术研讨会 https://github.com/PaddlePaddle/PaddleOCR/issues/10223 讨论，我们确定了XXXX任务。

## 解决步骤
1. 根据开源代码进行网络结构、评估指标转换。代码链接：XXXX
2. 结合[论文复现指南](https://github.com/PaddlePaddle/models/blob/release%2F2.2/tutorials/article-implementation/ArticleReproduction_CV.md)，进行前反向对齐等操作，达到论文Table.1中的指标。
3. 参考[PR提交规范](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_ch/code_and_doc.md)提交代码PR到ppocr中。

看到百度ai开放平台有混贴票据识别的项目，请问开源这边有没有对应的项目？
或者，指点一下如何实现？

**Comment by GreatV** [comment]
可以参考
- https://github.com/PaddlePaddle/PaddleOCR/blob/main/applications/%E5%A4%9A%E6%A8%A1%E6%80%81%E8%A1%A8%E5%8D%95%E8%AF%86%E5%88%AB.md
- https://aistudio.baidu.com/global/search?keyword=%E7%A5%A8%E6%8D%AE&tab=ALL

**Reply by greatliu** [reply]
这里面并没有混贴的识别啊，我的意思是，好多种票据贴到一张纸上，都能识别出来。
比如把火车票和发票贴一起。

**Comment by BugManL** [comment]
是不是可以用目标检测的方式，比如你先有一堆发票扫描件和其他票据扫描件，然后拼成A4的图片然后用于目标检测训练，再根据训练后的模型用于检测一张纸的上出现的票据，并带上bbox，方便你裁切出来做二次处理
